doc/src/devdocs/build/windows.md

```diff
@@ -166,8 +166,8 @@ Note: MSYS2 requires **64 bit** Windows 7 or newer.
 
     2. Clone the Julia sources:
 
-        ```
-        git clone https://github.com/JuliaLang/julia.git
+        ```sh
+        git clone https://github.com/JuliaLang/julia.git &&
```

Comment:
IMO this `&&` doesn't increase clarity.

Reply:
Incidentally this does not happen in the `Cygwin` section

Reply:
hahahaha turns out this is because of incorrect spacing. This is the render after putting in the correct number of spaces: 
<img width="817" alt="image" src="https://github.com/user-attachments/assets/62d3002d-f6fc-4d64-8d3f-91a78709d381" />

This is kinda lame to be honest. I dont know why markdown is so sensitive to spaces.